src/assets/wise5/authoringTool/components/component-selector/component-selector.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ComponentFactory } from '../../../common/ComponentFactory';
+import { MatDialog } from '@angular/material/dialog';
+import { ComponentInfoService } from '../../../services/componentInfoService';
+import { ComponentInfoDialogComponent } from '../component-info-dialog/component-info-dialog.component';
+
+@Component({
+  selector: 'component-selector',
+  templateUrl: './component-selector.component.html',
+  styleUrls: ['./component-selector.component.scss']
+})
+export class ComponentSelectorComponent {
+  @Output() componentSelectedEvent: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
No need to emit any value since this selector is already for a specific component type?
```suggestion
  @Output() componentSelectedEvent: EventEmitter<void> = new EventEmitter<void>();
```